frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -180,6 +183,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
   },
+  {
+    type: 'OCSKebabActions',
```

Comment:
We should add an RBAC check

Reply:
This should be

```suggestion
    type: 'PersistentVolumeClaim/Action',
```

to match `ClusterServiceVersion/Action` (with a similar variable names throughout)

Reply:
As communicated by @Madhu-1, RBAC will be added during csi deployment, user should not worry about any RBAC. Even if it is required, we will send it as a separate PR later.

Reply:
I will address the RBAC feature in a separate PR as I want to get this PR merged soon as this is dependent for snapshot PR.